src/app/validators/numbers-only.validator.ts

```diff
@@ -0,0 +1,11 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+import { Validation } from '../layouts/base/pages/components/transactions-table/transactions-table.constants';
+
+export function numbersOnlyValidator(): ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
```

Comment:
bad aproach, antipattern, in ngative case make return empty array -> (control: AbstractControl): ValidationErrors